presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -125,7 +125,8 @@ public synchronized Response upgrade(@ApiParam("Url to fetch package") String ur
             LOGGER.error(e, "Invalid url: %s", urlToFetchPackage);
             return Response.status(BAD_REQUEST).entity("Invalid url").build();
         }
-        return Response.status(ACCEPTED).entity("Presto is being upgraded.").build();
+        return Response.status(ACCEPTED).entity("Presto is being upgraded.\r\n" +
```

Comment:
This message is updated since presto version can be obtained now if presto is not running